featurebyte/service/table_facade.py

```diff
@@ -175,6 +175,15 @@ async def update_table_status(
         await self.table_status_service.update_status(
             service=service, document_id=table_id, status=status
         )
+        if status == TableStatus.DEPRECATED:
+            # remove entity tagging from deprecated table
+            table = await service.get_document(table_id)
+            columns_info = []
+            for col_info in table.columns_info:
+                if col_info.entity_id:
+                    col_info.entity_id = None
+                columns_info.append(col_info)
+            await self.update_table_columns_info(table_id, columns_info, service=service)
```

Comment:
Do we need to also update `table_ids` in the entity model?

Reply:
This method in the facade class will delegate it to table column info service, it is the same effect as making api call to untag the entity from the table:
https://github.com/featurebyte/featurebyte/blob/main/featurebyte/service/table_columns_info.py